docsrc/conf.py

```diff
@@ -28,7 +28,7 @@
 author = 'FMR LLC'
 
 # The full version, including alpha/beta/rc tags
-release = '1.2.1'
+release = '1.3.1'
```

Comment:
1.3.0 would make more sense.

Reply:
We actually had a `1.3.0`, my bad for not updating the docs!